src/rgw/services/svc_bi_rados.cc

```diff
@@ -31,20 +31,25 @@ void RGWSI_BucketIndex_RADOS::init(RGWSI_Zone *zone_svc,
 
 int RGWSI_BucketIndex_RADOS::open_pool(const rgw_pool& pool,
                                        RGWSI_RADOS::Pool *index_pool,
-                                       bool mostly_omap)
+                                       bool mostly_omap, const Span& parent_span)
```

Comment:
ditto, suppressing further examples

Reply:
@mattbenjamin  should I do those to helper functions too

Reply:
depends if their origin is clear, but most likely, yes--especially if you shorten the namespace

Reply:
okay